### PR TITLE
fix: cancel request example import

### PR DESCRIPTION
**Pull request by SoftwareAndOutsourcing**
Fixes cancel request example import.

### CODE REVIEW

**Comment by jimmywarting** [comment]
Hmm, wondering if not we should try to fix the real problem that is that we return a custom AbortError message, it should rather be a `DOMException` 

```js
// Browser test:
ctrl = new AbortController() 
ctrl.abort()
fetch('/', ctrl).catch(console.log) 
// DOMException { 
//   name: AbortError,
//   code: 20,
//   message: "Failed to execute 'fetch' on 'Window': The user aborted a request." 
// }
```

We could optionally start to depend on [node-domexception](https://www.npmjs.com/package/node-domexception). it's already included by another sub dependency of ours (fetch-blob).

File: README.md [hunk 1]
```diff
@@ -470,7 +470,7 @@ You may cancel requests with `AbortController`. A suggested implementation is [`
 An example of timing out a request after 150ms could be achieved as the following:
 
 ```js
-import fetch from 'node-fetch';
+import fetch, { AbortError} from 'node-fetch';
```
Comment (LinusU) [inline README.md:473]:
```suggestion
import fetch, { AbortError } from 'node-fetch';
```

**Review by LinusU — APPROVED** [review]
LGTM 👍

**Comment by LinusU** [comment]
@jimmywarting I think that would be nice, but I still think that we should merge this one in the mean time 👍

**Comment by jimmywarting** [comment]
Alright. maybe change it in next major release to avoid breakage?

**Comment by jimmywarting** [comment]
should maybe have done `docs:` instead of `fix:` to avoid bumping an unnecessary release

**Pull request merged**